be/src/vec/data_types/data_type.cpp

```diff
@@ -56,6 +56,7 @@ void IDataType::update_avg_value_size_hint(const IColumn& column, double& avg_va
 
 ColumnPtr IDataType::create_column_const(size_t size, const Field& field) const {
     auto column = create_column();
+    column->reserve(size);
```

Comment:
const column nested column only need size == 1, do not need reserve `size`

Reply:
Use `reserve(1)` instead.